Density/app/src/main/res/layout/facility_info_page.xml

```diff
@@ -137,14 +137,113 @@
                 android:layout_height="wrap_content"
                 android:layout_marginTop="8dp"
                 android:layout_marginEnd="16dp"
-                android:layout_marginBottom="16dp"
                 android:gravity="end"
                 android:text="@string/accuracy"
                 android:textSize="12sp"
                 android:textStyle="italic"
                 app:layout_constraintEnd_toEndOf="parent"
                 app:layout_constraintTop_toBottomOf="@+id/current_view" />
 
+            <HorizontalScrollView
```

Comment:
When testing on Nexus 5 emulator, this doesn't look good/behave as expected - should fix before merge.